docs/agentd/agent_CLI.md

```diff
@@ -0,0 +1,17 @@
+# Building the Agent
+
+1. From within the cmd/aries-agentd directory, run `go build`.
```

Comment:
go build can be done by passing output location, so refrain from using second bullet point.

show use of `-o` in examples and run from there

Reply:
Done